openstack/blockstorage/extensions/volumeactions/results.go

```diff
@@ -50,6 +55,101 @@ func (r commonResult) Extract() (map[string]interface{}, error) {
 	return s.ConnectionInfo, err
 }
 
+// VolumeImage contains information about volume upload to an image service.
```

Comment:
Yeah, that's a good point.

So the result is information about a volume which has just turned into an image. I like `ImageVolumeType`. It keeps `VolumeType`, which I think is important, and the `Image` part easily conveys that it's restricted to just the uploaded image.

Thank you :)